src/plugins/dashboard/public/application/top_nav/show_share_modal.tsx

```diff
@@ -104,21 +111,45 @@ export function ShowShareModal({
     );
   };
 
+  const rawDashboardState = stateToRawDashboardState({
+    state: currentDashboardState,
+    version: kibanaVersion,
+  });
+
+  const locatorParams: DashboardAppLocatorParams = {
+    ...rawDashboardState,
+    forwardStateToHistory: true,
```

Comment:
forwardStateToHistory and options are not part of `DashboardAppLocatorParams` interface

Reply:
also i don't see this used anywhere ?

Reply:
Yeah, this should be removed!